lib/src/test/resources/src/account.cairo

```diff
@@ -0,0 +1,379 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Can we do `oz_account/{contract.cairo, license}`?

Reply:
Maybe `oz_contracts` since it's both account and deployer? This however complicates our compilation scripts.

Reply:
however you want, just structure it a little bit. It can also be account_oz.cairo and account_oz_license